### DISCUSSION TITLE
Apache configuration for SSP

### DISCUSSION THREAD

**Discussion by ghost** [ltb-project/self-service-password#560] [Q&A] [answered]
I'm trying to follow the instructions for setting up SSP on an Ubuntu test VM. I followed the [Ubuntu instructions](https://self-service-password.readthedocs.io/en/latest/installation.html#debian-ubuntu) with ease and Apache did install because I can see the default test page if I access the server's IP in a web browser. But now I am stumped on what I need to do to make SSP load instead of the default web page?

Could someone help me with this or point me in the right direction on what I need to edit? 🙏🏻

**Comment by faust64** [comment, marked as answer]
Hi,

Have you looked at the Webserver Configuration / Apache Configuration chapter?
https://self-service-password.readthedocs.io/en/v1.4/config_apache.html

There's a sample virtualhost - with no SSL, very basic, ...
Though if you're not using VirtualHosts, the main takeaway is to set your `DocumentRoot` to `/usr/local/self-service-password/htdocs`

**Reply by ghost** [reply]
Yes, I am not using virtual hosts. Thanks for pointing me in the direction of `DocumentRoot`. I was able to get SSP loading now by changing DocumentRoot in the default conf files. 😊
